Calibration/EcalCalibAlgos/src/miscalibExample.cc

```diff
@@ -31,36 +29,12 @@
 #include <stdexcept>
 #include <vector>
 
-// class declaration
-//
-/*
-class miscalibExample : public edm::EDAnalyzer {
-   public:
-      explicit miscalibExample(const edm::ParameterSet&);
-      ~miscalibExample();
-
-      virtual void analyze(const edm::Event&, const edm::EventSetup&);
-      virtual void beginJob();
-      virtual void endJob();
-   private:
-
-
-      // ----------member data ---------------------------
-      std::string rootfile_;
-      std::string correctedHybridSuperClusterProducer_;
-      std::string correctedHybridSuperClusterCollection_;
-      std::string BarrelHitsCollection_;
-      std::string ecalHitsProducer_ ;
-      int read_events;
-
-      TH1F* scEnergy;
-};
-
-*/
 miscalibExample::miscalibExample(const edm::ParameterSet& iConfig) {
   rootfile_ = iConfig.getUntrackedParameter<std::string>("rootfile", "ecalSimpleTBanalysis.root");
   correctedHybridSuperClusterProducer_ = iConfig.getParameter<std::string>("correctedHybridSuperClusterProducer");
-  correctedHybridSuperClusterCollection_ = iConfig.getParameter<std::string>("correctedHybridSuperClusterCollection");
+  edm::InputTag correctedHybridSuperClusterTag_ = iConfig.getParameter<edm::InputTag>("correctedHybridSuperClusterCollection"); 
```

Comment:
In this version of the file I removed the declaration of the `correctedHybridSuperClusterCollection` variable altogether because I wasn't sure how it was used aside from being an argument in the `getByLabel()` method. Should it still be there?

Reply:
Now I see your point - updated

Reply:
This line can actually be removed now. `correctedHybridSuperClusterTag_` is not used anymore.